kconfig_hardened_check/__init__.py

```diff
@@ -380,6 +380,8 @@ def construct_checklist(l, arch):
     if arch in ('X86_64', 'ARM64', 'X86_32'):
         stackleak_is_set = OptCheck('self_protection', 'kspp', 'GCC_PLUGIN_STACKLEAK', 'y')
         l += [stackleak_is_set]
+        l += [OR(OptCheck('self_protection', 'kspp', 'RANDOMIZE_KSTACK_OFFSET_DEFAULT', 'y'),
```

Comment:
Hello @anthraxx, thanks for your pull request!

I'll merge it. Please, fix two issues:
1. Change `kspp` to `my` and move this check to `# 'self_protection', 'my'` section. We should do that because [KSPP Recommended Settings](http://kernsec.org/wiki/index.php/Kernel_Self_Protection_Project/Recommended_Settings) currently don't contain this option.
2. Remove the version check. It is used for options that are enabled by default starting from some kernel version.

Thanks!
Please force-push your branch.

Best regards,
Alexander

Reply:
Ah, yes.
And please add `randomize_kstack_offset=1` to the comment `# N.B Hardening command line parameters`.
Thanks!